schema/Connection.re

```diff
@@ -1,9 +1,56 @@
-type pageInfo = {
-  startCursor: option(string),
-  hasPreviousPage: bool,
-  hasNextPage: bool,
-  endCursor: option(string),
-  total: option(int),
+open Graphql_lwt;
```

Comment:
can you put this under a folder called graphql-helpers? we'll definitely be moving this to other projects

Reply:
okay